dom.go

```diff
@@ -268,8 +271,16 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 // function is not used directly but rather the elem subpackage (which is type
 // safe) is used instead.
 func Tag(tag string, m ...MarkupOrComponentOrHTML) *HTML {
+	return NamespacedTag(``, tag, m...)
```

Comment:
I've no problem changing this for consistency, but as a general personal query, can you explain the assertion that interpreted strings are simpler?

I tend to use raw strings liberally, but I may examine that behaviour if there are technical deficiencies that I'm unaware of.